clinica/pipelines/t1_freesurfer_atlas/t1_freesurfer_atlas_utils.py

```diff
@@ -21,45 +25,87 @@ def compute_atlas(
     for path in Path(path_to_atlas).rglob(
         "*" + to_process_with_atlases[0] + "_6p0.gcs"
     ):
-        hemisphere = Path(path.stem).stem
-        atlas_name = Path(path.stem).suffix[1:].split("_")[0]
-        sub, ses = to_process_with_atlases[1].split("_")
-        subject_dir = (
-            caps_directory
-            + "/subjects/"
-            + sub
-            + "/"
-            + ses
-            + "/t1/freesurfer_cross_sectional"
-        )
+        # if the length of the split to_process_with_atlases[1] is 2, that means the format is sub-X_ses-Y
```

Comment:
This comment is out-of-date with the code I think?

Reply:
Yes it is, since the if condition was updated. I will remove it.